apple-pie/styles/style.css

```
@@ -21,4 +21,44 @@ body {
 }
 
 /* STYLES */
-/* Write your CSS below */
\ No newline at end of file
+/* Write your CSS below */
+  
+
+header {
+  background-image: url(/apple-pie/images/apple-pie.jpg);
```

```suggestion
  background-image: url(../images/apple-pie.jpg);
```
En este caso el css esta dentro de la carpeta styles, entonces tenemos que salirnos una carpeta y ya si estariamos como si estuvieramos en apple-pie! :D (por cierto bien usado el url!!!!!)